test/testing_utils.dart

```diff
@@ -0,0 +1,224 @@
+    }
+    print("dismissOverlay: dismissed cutout-shadow-overlay.");
+    await waitForAngular(driver);
+  }
+}
+
+Future<bool> isLoggedIn(WebDriver driver) async {
+  var profilePicture = null;
+  bool loggedIn = false;
+  try {
+    await waitForAngular(driver);
+    profilePicture = await driver.findElement(const By.cssSelector(
+        'nav > div > div > div.menu-item--profile > a > account-picture'));
+  } on NoSuchElementException catch (e) {
+    print("isLoggedIn: Logged-in-profile element not found.");
+  } catch (exception, stackTrace) {
+    print("Exceptions caught");
+    print(exception);
+    print(stackTrace);
+  } finally {
+    loggedIn = elementExists(profilePicture);
+    return loggedIn;
+  }
+}
+
+Future logoutUser(WebDriver driver) async {
+  bool loggedIn = await isLoggedIn(driver);
+  var profileBtn;
+  var logoutBtn;
+  if (loggedIn) {
+    try {
+      await waitForAngular(driver);
+      profileBtn = await driver.findElement(
+          const By.cssSelector('div.menu-item--profile > a > account-picture'));
+      logoutBtn = await driver.findElement(const By.xpath(
+          '*//div[contains(@class, "menu-item")]/a/span[contains(text(), "Kilépés")]'));
+    } on NoSuchElementException catch (e) {
+      print("logoutUser: Logout element(s) not found.");
+    } catch (exception, stackTrace) {
+      print("Exceptions caught:");
+      print(exception);
+      print(stackTrace);
+    } finally {
+      await waitForAngular(driver);
+      if (elementExists(profileBtn) && elementExists(logoutBtn)) {
+        await profileBtn.click();
+        await logoutBtn.click();
+      }
+      print("logoutUser: Logged out");
+    }
+  } else {
+    print("logoutUser: User wasnt logged in");
+  }
+}
+
+Future clickMegvasarlom(WebDriver driver, String productSlug) async {
+  await navigateToPage(driver, productSlug);
+  await waitForAngular(driver);
+  await sleep5();
+  var btnBuy = await driver.findElement(const By.xpath(
+      "*//div[contains(@class, 'package-details')]/material-button/material-ripple"));
+  await btnBuy.click();
+  await waitForAngular(driver);
+  await sleep5();
+}
+
+Future<bool> checkForConsoleErrors(WebDriver driver, String logLevel) async {
+  bool hasErrors = false;
+  await driver.logs
+      .get('browser')
+      .where((LogEntry l) => l.level == logLevel)
+      .toList()
+      .then((logs) {
+    if (logs.length > 0) {
+      logs.forEach(print);
+      hasErrors = true;
+    }
+  });
+  return hasErrors;
+}
+
+Future sleep10() {
+  return new Future.delayed(const Duration(seconds: 10), () => "10");
+}
+
+Future sleep5() {
+  return new Future.delayed(const Duration(seconds: 5), () => "5");
+}
+
+Future sleep2() {
+  return new Future.delayed(const Duration(seconds: 2), () => "2");
+}
+
+bool futureElementExists(Stream<WebElement> e) {
+  return (e != null);
+}
+
+bool elementExists(WebElement e) {
+  return (e != null);
+}
+
+bool isXAtLeastValueOfY(num x, num y) {
```

Comment:
There are build in matchers for numeric ordering (and a lot of other comparisons). This one is called `Matcher greaterThan(value)`